core/src/test/java/io/grpc/internal/RetriableStreamTest.java

```diff
@@ -147,6 +147,27 @@ public double nextDouble() {
   private final ChannelBufferMeter channelBufferUsed = new ChannelBufferMeter();
   private final FakeClock fakeClock = new FakeClock();
 
+  private static double calculateJitterFactor() {
```

Comment:
use the `RetriableStream.intervalWithJitter` I recommended above.  As a general note be careful of your names really meaning what the function does.  In this case, the dictionary version of "jitter factor" is actually  a value in the range [-0.2,0.2]  which is (`1 - <your calculated value>`). This was confusing until I realized what you had done.